backend/protocol_rpc/endpoints.py

```
@@ -791,26 +791,95 @@
+                        transaction_data,
+                        value,
+                        genlayer_transaction.type.value,
+                        retry_nonce,  # Use fresh nonce calculation
+                        leader_only,
+                        genlayer_transaction.max_rotations,
+                        None,
+                        None,  # Force hash regeneration
+                        genlayer_transaction.num_of_initial_validators,
+                    )
+                else:
+                    raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)